app/main/views/sign_in.py

```diff
@@ -171,3 +181,22 @@ def sign_in():
 @login_manager.unauthorized_handler
 def sign_in_again():
     return redirect(url_for("main.sign_in", next=request.path))
+
+
+def check_for_gov_email_address(user_email):
+    # We could try to check that it is a government email at the time the invite is
+    # sent, but due to the way login.gov allows multiple emails, it would not be effective.
+    # We track the login.gov user by their uuid, so if they have a login.gov account
+    # with a .gov email address and a .com email address, the .com address will work without
+    # having a check here.
+    if (
+        user_email.lower().endswith(".gov")
+        or user_email.lower().endswith(".mil")
+        or user_email.lower().endswith(".si.edu")
+    ):
+        # everything is good, proceed
+        pass
+    else:
+        current_app.logger.warning("invited user has a non-government email address.")
+        flash("You must use a government email address.")
+        abort(403)
```

Comment:
Taking into account the note above about moving this method into the `app/utils/user.py` file, this would go away and just come from a new import instead.

```suggestion
```